packages/svelte2tsx/src/emitDts.ts

```diff
@@ -2,21 +2,25 @@ import * as path from 'path';
 import ts from 'typescript';
 import { svelte2tsx } from './svelte2tsx';
 
-export interface EmitDtsConig {
+export interface EmitDtsConfig {
     declarationDir: string;
     svelteShimsPath: string;
     libRoot?: string;
 }
 
-export async function emitDts(config: EmitDtsConig) {
+// to make typo fix non-breaking, continue to export the old name but mark it as deprecated
```

Comment:
The Deprecated interface needs to be added to the `index.d.ts` file, not this file. The `index.d.ts` contains all the public API.